sources/packages/web/src/components/generic/HeaderTitleValue.vue

```diff
@@ -20,7 +20,7 @@ export default defineComponent({
     },
     value: {
       type: String,
-      required: true,
+      required: false,
```

Comment:
Is it no longer required for the situation where the value is about to be loaded but the component is already displayed?

Reply:
<img width="685" alt="image" src="https://github.com/bcgov/SIMS/assets/7859295/55543289-bc9b-4d22-8e87-80ea6fa141bb">

Since `value` can be received as a `prop` or optionally as a `slot`, `required` is set to false. This is unlike the `title` which only receives it via `prop`